examples/fluids/navierstokes.c

```diff
@@ -677,8 +740,12 @@ int main(int argc, char **argv) {
   ierr = PetscOptionsString("-ceed", "CEED resource specifier",
                             NULL, ceedresource, ceedresource,
                             sizeof(ceedresource), NULL); CHKERRQ(ierr);
-  ierr = PetscOptionsBool("-test", "Run in test mode",
-                          NULL, test=PETSC_FALSE, &test, NULL); CHKERRQ(ierr);
+  testChoice = TEST_NONE;
```

Comment:
Would it be possible to initialize this to NULL so that we don't need the special TEST_NONE?

Reply:
`testChoice` is of type `testType`, which is an enum. So we could not assign `NULL` to an enum type. It would cause an assignment mismatch warning:
```
warning: incompatible pointer to integer conversion assigning to 'testType' from 'void *'
```